draft-ietf-opsawg-oam-characterization.xml

```diff
@@ -156,16 +163,34 @@ document does not modify the definitions in <xref target="RFC7799" />.
 
 
 
-<t hangText="Packet OAM:"> OAM in relation to a user data packet.</t>
+<t hangText="Active, Passive, Hybrid, and In-Packet OAM"></t>
+
+          <t>
+               <xref target="RFC7799" /> provides clear definitions for active and passive
+               performance assessment such that the construction of metrics and
+               methods can be described as either "Active" or "Passive".  Even
+               though <xref target="RFC7799" /> does not include the specific terms "Active",
+               "Passive", or "Hybrid" as modifiers of "OAM", the following terms
+               are used in many RFCs and are provided here for clarity.
+          
+            <list style="hanging">
+              <t hangText="Active OAM:"><br /> Depends on dedicated OAM packets.</t>
+              <t hangText="Passive OAM:"><br /> Depends solely on the observation of one
+      or more existing data packet streams and does not use dedicated OAM packets.</t>
+               <t hangText="Hybrid OAM:"><br /> Uses instrumentation or modification of data packets themselves. <xref target="RFC9341" /> and <xref target="RFC9197" /> are examples labeled "Hybrid OAM" under this definition.
+               </t>
+            </list>
+          </t>
```

Comment:
RFC9341 uses existing packet fields, while RFC9197 piggy-backs an OAM structure and fields.

Even if these cases do not have a new name, explaining how they are different seems important.